Gemfile

```diff
@@ -1,4 +1,9 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
+
+# Ruby Version
 ruby '3.1.0'
 
-gem "rspec"
+# Gems
+gem 'rspec'
+gem 'rubocop', require: false
+gem 'simplecov', require: false, group: :test
```

Comment:
You can organize your gemfile groups as bellow:

```rb
group :test do
  gem 'simplecov', require: false
end
``` 
It will be easier to read your Gemfile in the future.

This link has some recommendations about it: [Bundler and Gemfile best practices](https://depfu.com/blog/2017/01/18/bundler-and-gemfile-best-practices#use-gemfile-groups)

What do you think?

Reply:
Interesting! I will follow your recommendation.